docs/framework/get-started/index.md

```diff
@@ -80,7 +80,7 @@ The .NET Framework is a runtime execution environment that manages applications
 |[System Requirements](../../../docs/framework/get-started/system-requirements.md)|Lists the hardware and software requirements for running the .NET Framework.|  
 |[.NET Core and Open-Source](../../../docs/framework/get-started/net-core-and-open-source.md)|Describes what .NET Core is in relation to the .NET Framework and how to access the open-source .NET Core projects.|  
 |[.NET Core documentation](https://docs.microsoft.com/dotnet/)|The conceptual and API reference documentation for .NET Core.|  
-|[Installation Guide](../../../docs/framework/install/guide-for-developers.md)|Provides information about installing the .NET Framework.|  
+|[Install the .NET Framework for developers](../../../docs/framework/install/guide-for-developers.md)|Provides information about installing the .NET Framework.|  
   
 ## See Also  
  [.NET Framework 4.6 and 4.5](../../../docs/framework/index.md)   
```

Comment:
maybe for another PR but this string (.NET Framework 4.6 and 4.5) should be removed from the docs - I've seen it a few times. We should now call it .NET Framework Guide IMO.